.github/workflows/run-tests.yml

```
@@ -3,14 +3,40 @@ run-name: "Run Tests: ${{ github.ref_type }} ${{ github.ref_name }} by ${{ githu
 
 jobs:
+  lint:
+    name: Lint translator
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Set up Python ${{ matrix.python-version }}
```

actionlint (1.7.8)

property "python-version" is not defined in object type {}

(expression)